Hunting Queries/Microsoft 365 Defender/General queries/Endpoint Linux Agent Health Status Report

```diff
@@ -0,0 +1,28 @@
+
+name: Endpoint Linux Agent Health Status Report
+description: |
+    This query will provide a report on Linux configurations for Microsoft Defender Endpoint Protection.
+    Tests which are reporting "BAD" as a result imply that the associated capability is not configured per best practice recommendation.
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceTvmSecureConfigurationAssessment
+tactics:
+- Configuration
+- Linux Agent Health
+query: |
+DeviceTvmSecureConfigurationAssessment
+| where OSPlatform == "Linux"
+| where ConfigurationId in ('scid-6001', 'scid-6002', 'scid-6090', 'scid-6091', 'scid-6094', 'scid-6095')
+| extend Ted = case(
+    ConfigurationId == "scid-6001", "SensorDataCollection",
+    ConfigurationId == "scid-6002", "ImpairedCommunications",
+    ConfigurationId == "scid-6095", "AntivirusSignatureVersion",
+    ConfigurationId == "scid-6090", "RealtimeProtection",   
+    ConfigurationId == "scid-6091", "PUAProtection",
+    ConfigurationId == "scid-6094", "CloudProtection",
+    "N/A"),
+    Result = case(IsApplicable == 0, "N/A", IsCompliant == 1, "GOOD", "BAD")
+| extend packed = pack(Ted, Result)
+| summarize Tests = make_bag(packed), DeviceName = any(DeviceName) by DeviceId
+| evaluate bag_unpack(Tests)
```

Comment:
Ideally, we should also have an entity mapping field in the end.  
Read more about it here: https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#entity-mapping